config/locales/titles/en.yml

```diff
@@ -10,9 +10,10 @@ en:
       password: Edit your password
       phone: Edit your phone number
     enter_2fa_code: Enter the secure one-time security code
-    loa3_verified:
-      'true': You have verified your identity with %{app}
-      'false': You have created your account with %{app}
+    loa3_verified_html: You have %{accent} with %{app}
+    loa3_verified_accent:
```

Comment:
Also, does this page show if the user already has an account but is going to a particular SP for the first time? (That's an explicit goal of the attributes page, it might not be handled in this PR, I'm just trying to understand how it all fits in)